src/resources/SchemaService/SchemaServiceInterfaces.ts

```diff
@@ -31,11 +31,15 @@ export interface SchemaServiceQueryParams {
     username?: string;
 }
 
-export interface OffsetOrLimit {
+export interface GetEntitiesQueryParams extends SchemaServiceQueryParams {
```

Comment:
This is a breaking change since the interface name changed, please edit your commit message accordingly